content/practices/team-sentiment.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Team Sentiment"
+date: 2018-03-31T22:53:43+01:00
+draft: false
+---
+
+
+## What is it?
+
+Team Sentiment practices track the mood of a team and provide a mechanism for team members to rapidly feed back, either publically or anonymously, when their mood has changed.
+
+## Why use a Team Sentiment pratice?
+
+Team sentiment practices enable problems to be identified, radiated and addressed early. They enable the team to quickly discuss where one or more team members is suddenly troubled by the direction their work is going and can also provide an information radiator to assess the overall team health and feeling. 
+
+
+## Related Practices
+
+Team sentiment practices are often introduced where there is a strong sense of servant leadership in the working environment. 
```

Comment:
I like this sentence, but not in this section. I feel it is a great addition to WHY we would want to use Team Sentiment.